test/integration/basic_test.go

```diff
@@ -181,11 +179,19 @@ func FeatureContext(s *godog.Suite) {
 }
 
 func TestMain(m *testing.M) {
+	flag.StringVar(&givenArgs, "minishift-args", "", "Arguments to pass to minishift")
+	flag.StringVar(&givenPath, "binary", "", "Path to minishift binary")
+	flag.StringVar(&givenTags, "tags", "", "Tags of tests to pass to godog")
+	flag.Parse()
+
+	givenTags = strings.Replace(givenTags, " ", ", ", -1)
```

Comment:
@hferentschik If no tags are given `make integration TEST_TAGS=""` then all tests are executed. Currently only problem I have found is this case: `make integration TEST_TAGS="basic openshift "` with extra space or comma at the end it fails.